hbase-server/src/main/java/org/apache/hadoop/hbase/rsgroup/RSGroupBasedLoadBalancer.java

```diff
@@ -234,6 +237,9 @@ public ServerName randomAssignment(RegionInfo region,
       if (!fallbackRegions.isEmpty()) {
         List<ServerName> candidates = null;
         if (isFallbackEnabled()) {
+          LOG.debug("Falling back {} regions to servers outside their RSGroup. Regions: {}",
```

Comment:
Better wrap it with a LOG.isDebugEnabled? We have a stream operation when constructing the parameter.